### ISSUE TITLE
Add small amount of margin/padding to desktop navbar

### ISSUE THREAD

**Issue by kurealnum** [toddagriscience/nightcrawler#334]
### Open a feature request issue

The desktop navbar is a little bit too tall right now, and should be have a bit of margin/padding added to the top for sake of accessibility (it's a little bit too close to the bookmarks/tab bar). How much you add is up to you, but aim for somewhere between 4px and 16px.
